tests/macsec/test_deployment.py

```diff
@@ -23,3 +26,45 @@ def test_config_reload(self, duthost, ctrl_links, policy, cipher_suite, send_sci
         assert wait_until(300, 6, 12, check_appl_db, duthost, ctrl_links, policy, cipher_suite, send_sci)
         # Recover the original config file
         duthost.shell("sudo cp config_db.json /etc/sonic/config_db.json")
+
+    @pytest.mark.disable_loganalyzer
+    def test_scale_rekey(self, duthost, ctrl_links, rekey_period, wait_mka_establish):
```

Comment:
duthost will also be for the uplink card?

Reply:
Currently it takes the first card (uplink) by default for all macsec tests, I have a follow up PR to fix all . plan to use the fixture macsec_duthost.